tests/framework/Samplers/Restart/test_restart_csv.xml

```diff
@@ -23,14 +23,14 @@
 
   <Steps>
     <MultiRun name="make">
-      <Input class="DataObjects" type="PointSet">solns</Input>
+      <Input class="DataObjects" type="PointSet">restart</Input>
       <Model class="Models" type="ExternalModel">fine</Model>
       <Sampler class="Samplers" type="MonteCarlo">mc</Sampler>
       <Output class="DataObjects" type="PointSet">solns</Output>
     </MultiRun>
     <IOStep name="load">
       <Input class="Files" type="">csv</Input>
-      <Output class="DataObjects" type="PointSet">solns</Output>
+      <Output class="DataObjects" type="PointSet">restart</Output>
```

Comment:
what will happen here? The ```restart``` is used as input in previous MultiRun, and used as output in this IOStep.

Reply:
In order of use, it is an output of this IOStep (to load data into) and as input of the MultiRun (to read data from).  The problem occurs when an object is the output of multiple MultiRun steps.

Reply:
got it.